src/scss/pages/_how-it-works.scss

```diff
@@ -162,3 +133,64 @@
     }
   }
 }
+
+//* Safety card container *//
+
+.safety {
+  > div {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    margin: 0 0 0 auto;
+    padding: 5rem;
+
+    // *Safety cards (3 horizontal cards each containing image, heading and short list) *//
+    > div {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-self: flex-start;
+      align-items: center;
+      padding: 2rem;
+
+      @include breakpoint-up("large") {
+        margin: 0 3rem;
+      }
+    }
+
+    > div > img {
+      height: 8rem;
+      width: 8rem;
+      padding: 0.5rem;
+      border-radius: 8rem;
+      background-color: var(--color-tertiary);
+    }
+
+    > div > div {
```

Comment:
`.safety > div > div > div` may indicate adjusting the markup would be useful or possibly a class. It's a little opaque to read. But I haven't seen the design or markup so it could be fine for the use case.

Reply:
Yeah this is kind of a legacy section, there are a few of them across the website and I will redo them whenever I start working on the grid system